src/amo/components/CollectionManager/index.js

```diff
@@ -154,7 +155,28 @@ export class CollectionManagerBase extends React.Component<Props, State> {
     event: ElementEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
     event.preventDefault();
-    this.setState({ [event.target.name]: event.target.value });
+    const { name, value } = event.target;
+    const { creating } = this.props;
+
+    if (value === null || typeof value === 'undefined') {
```

Comment:
I think the second part of this condition deserves a test too.

Reply:
I am not sure when are we expecting `event.target.value` to be `undefined` or `null`.
DO we have any case for that?

Reply:
Ah well, I don't know. If that never happens, we should not have this condition no?